runtime/src/main/java/org/corfudb/protocols/logprotocol/MultiObjectSMREntry.java

```diff
@@ -57,7 +45,9 @@ protected MultiSMREntry getStreamEntry(UUID streamID) {
      * @param updateEntry SMREntry to add
      */
     public void addTo(UUID streamID, SMREntry updateEntry) {
-        getStreamEntry(streamID).addTo(updateEntry);
+        List<SMREntry> updates = entryMap.getOrDefault(streamID, new ArrayList<>());
```

Comment:
Before this is committed, it will be owned by a transaction (i.e. contained in thread local storage), so adding to this collection should be thread safe.

Reply:
These read as pretty useful comments to me, any reason why they cannot be comments in the code itself?